via/views/decorators.py

```diff
@@ -35,3 +37,27 @@ def view_wrapper(context, request):
         return view(context, request)
 
     return view_wrapper
+
+
+def has_secure_url_token(view, signature_param="via.sec"):
+    """Intended to be used as a decorator for pyramid views.
+
+    Checks the validity of URL signature present in the `url_param` query parameter.
+
+    :param url_param Name of the query param containing the signature
+    """
```

Comment:
```suggestion
    """Require the request to have a valid signature in the "via.sec" query param."""
```

Why have the `signature_param` argument? I don't think the view config can actually use it

Reply:
It's not something you have a choice about any more